src/content/en/2021/cdn.md

```diff
@@ -1,202 +1,198 @@
+This reduction in requests reduces the load on the internet backbone, ([read Middle-Mile of the Internet](https://en.wikipedia.org/wiki/Middle_mile)). It also helps manage more of the internet load within the last mile of the internet. Thus, a CDN plays a multifaceted role in the Internet landscape as it allows web owners to improve the performance, reliability and scalability of content delivery.
 
-### Caveats and Disclaimers
+### Caveats and disclaimers
 
 As with any observational study, there are limits to the scope and impact that can be measured. The statistics gathered on CDN usage for the Web Almanac are focused more on applicable technologies in use and not intended to measure performance or effectiveness of a specific CDN vendor. While this ensures that we are not biased towards any CDN vendor, it also means that these are more generalized results.
 
 These are the limits to our testing [methodology](./methodology):
 
-**Simulated network latency:** We use a dedicated network connection that synthetically shapes traffic.
+- **Simulated network latency:** We use a dedicated network connection that synthetically shapes traffic.
 
-**Single geographic location:** Tests are run from a single datacenter and cannot test the geographic distribution of many CDN vendors.
+- **Single geographic location:** Tests are run from a single datacenter and cannot test the geographic distribution of many CDN vendors.
 
-**Cache effectiveness:** Each CDN uses proprietary technology and many, for security reasons, do not expose cache performance.
+- **Cache effectiveness:** Each CDN uses proprietary technology and many, for security reasons, do not expose cache performance.
 
-**Localization and internationalization:** Just like geographic distribution, the effects of language and geo-specific domains are also opaque to these tests.
+- **Localization and internationalization:** Just like geographic distribution, the effects of language and geo-specific domains are also opaque to these tests.
 
-**CDN detection:** This is primarily done through DNS resolution and HTTP headers. Most CDNs use a DNS CNAME to map a user to an optimal datacenter. However, some CDNs use AnyCast IPs or direct A+AAAA responses from a delegated domain which hide the DNS chain. In other cases, websites use multiple CDNs to balance between vendors, which is hidden from the single-request pass of our crawler.
+- **CDN detection:** This is primarily done through DNS resolution and HTTP headers. Most CDNs use a DNS CNAME to map a user to an optimal datacenter. However, some CDNs use Anycast IPs or direct A+AAAA responses from a delegated domain which hide the DNS chain. In other cases, websites use multiple CDNs to balance between vendors, which is hidden from the single-request pass of our crawler.
 
-All of this limits the effectiveness in the measurements.
+All of this influences our measurements.
```

Comment:
I toned this down a litte. Hope you don't mind @Navaneeth-akam

Reply:
Yup....looks good.